components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml

```diff
@@ -86,7 +86,6 @@
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/password_field"
-            style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
```

Comment:
Should we use `MozacPromptLoginTextInputLayoutCursorAppearance` style here? I don't see how they linked :)

Reply:
Oh I see it gets its value from the parent that has the style `MozTextInputLayout`, right?

Reply:
Right, from the parent.